Cargo.toml

```diff
@@ -82,7 +82,7 @@ content_inspector = { version = "0.2" }
 crc32c = { version = "0.6"}
 criterion = "0.5.0"
 crossbeam-channel = "0.5"
-current_platform = { version = "0.2" }
+current_platform = { workspace = true }
```

Comment:
maybe this line changed is not needed? not sure about the behavior of using this in the workspace Cargo.toml

Reply:
oh, true! I didn't realize I did it, thanks.
til that cargo accepts that